src/utils/auth.ts

```diff
@@ -57,7 +57,7 @@ export type AuthenticationStatus =
 
 const PORT = 42224;
 const AUTHORIZATION_TIMEOUT = 60_000;
-const AUTH0_SCOPES = ['profile', 'offline_access'] as const;
+const AUTH0_SCOPES = ['profile', 'offline_access', 'email'] as const;
```

Comment:
The frontend also includes `openId`, do we need that?

Reply:
This does too if you scroll down to where it's implemented.